action.yml

```diff
@@ -15,20 +18,72 @@ runs:
     - name: Install Python and depends
       run: |
         sudo apt install python3 python3-requests
+        cd ${{ github.action_path }}
+        pip install -r requirements.txt
       shell: bash
 
     - name: Run all PR best practice checks
+      if: github.event_name == 'pull_request_target'
       env:
-        BODY: ${{ github.event.pull_request.body }}
-        TITLE: ${{ github.event.pull_request.title }}
-        TOKEN: ${{ inputs.token }}
         REPOSITORY: ${{ github.event.repository.full_name }}
         PR_NUMBER: ${{ github.event.pull_request.number }}
+        PR_TITLE: ${{ github.event.pull_request.title }}
+        PR_BODY: ${{ github.event.pull_request.body }}
+        GITHUB_TOKEN: ${{ inputs.token }}
       run: |
-        python3 "${{ github.action_path }}/pr_best_practices.py" --pr-description "$BODY"
+        python3 "${{ github.action_path }}/pr_best_practices.py" --pr-description "$PR_BODY"
         set +e
-        python3 "${{ github.action_path }}/pr_best_practices.py" --pr-title "$TITLE"
+        python3 "${{ github.action_path }}/pr_best_practices.py" --pr-title "$PR_TITLE"
         if [ "$?" == 0 ]; then
-          python3 "${{ github.action_path }}/pr_best_practices.py" --add-label --token "$TOKEN" --repository "$REPOSITORY" --pr-number "$PR_NUMBER"
+          python3 "${{ github.action_path }}/pr_best_practices.py" --add-label --token "$GITHUB_TOKEN" --repository "$REPOSITORY" --pr-number "$PR_NUMBER"
         fi
       shell: bash
+
+    - name: Check for /jira command in PR comments
+      if: github.event_name == 'issue_comment' && github.event.issue.pull_request != null
+      env:
+        REPOSITORY: ${{ github.event.repository.full_name }}
+        PR_NUMBER: ${{ github.event.issue.number }}
+        PR_TITLE: ${{ github.event.issue.title }}
+        PR_BODY: ${{ github.event.issue.body }}
+        PR_AUTHOR: "${{ github.event.issue.user.login }}"
+        COMMENT_BODY: ${{ github.event.comment.body }}
+        GITHUB_TOKEN: ${{ inputs.token }}
+        JIRA_TOKEN: ${{ inputs.jira_token }}
+      run: |
+        if [[ "$COMMENT_BODY" =~ ^/jira-epic ]]; then
+          echo "🟢 Slash command '/jira-epic' detected"
+
+          # Exit if PR title contains a Jira ticket
+          echo "Bail if the pull request title already contains a Jira reference."
+          set +e
```

Comment:
This feels ugly, maybe let's use [something like this](https://stackoverflow.com/questions/18621990/bash-get-exit-status-of-command-when-set-e-is-active) for the next line?